fedimint-build/src/lib.rs

```diff
@@ -66,7 +66,7 @@ fn set_code_version_inner() -> Result<(), String> {
         // state by replacing the middle with 0s. This should
         // be noticeable enough, while letting find out the
         // root commit anyway.
-        format!("{}00000000{}", &hash[0..16], &hash[26..40])
+        format!("{}00000000{}", &hash[0..16], &hash[(40 - 16)..40])
```

Comment:
maybe `16+8` instead of 40

Reply:
Yeah, I agree this would be easier to reason about at a glance if it were

```
[0..16], [(16+8)..40]
```

or even

```
let zeros_count = 8;
format!(
    "{}{}{}",
    &hash[0..16],
    "0".repeat(zeros_count),
    &hash[(16 + zeros_count)..40]
)
```

Reply:
The damage is done, no need to overthink it. When it was as a constant (incorrect at at) no that it isn't, there's 10 ways to express it, and bikesheding starts. :D